mysql-test/suite/mariabackup/data_directory.test

```diff
@@ -21,4 +21,19 @@ rmdir $table_data_dir;
 SELECT * FROM t;
 DROP TABLE t;
 rmdir $targetdir;
+
+--echo #
+--echo # MDEV-18200 MariaBackup full backup failed with InnoDB: Failing assertion: success
+--echo #
+let $DATADIR= `select @@datadir`;
+chmod 0000 $DATADIR/ibdata1;
+--disable_result_log
+--error 1
+exec $XTRABACKUP --defaults-file=$MYSQLTEST_VARDIR/my.cnf --backup --target-dir=$targetdir;
+--enable_result_log
+chmod 0755 $DATADIR/ibdata1;
```

Comment:
I see that `have_symlink.inc` prevents this test from executing on Microsoft Windows. Hence, the `chmod` should not be an issue.

Reply:
`chmod` is a mtr command too.

Reply:
sorry @vaintroub missed your comment - testing with #2771

Reply:
Seems to be ignored.
```
--- D:/Buildbot/amd64-windows/build/mysql-test/suite/mariabackup/data_directory.result	2023-09-26 02:36:32.000000000 +0200
+++ D:\Buildbot\amd64-windows\build\mysql-test\suite\mariabackup\data_directory.reject	2023-09-26 02:42:42.841168000 +0200
@@ -1,4 +1,6 @@
 CREATE TABLE t(a INT) ENGINE=InnoDB DATA DIRECTORY='table_data_dir';
+Warnings:
+Warning	1618	<DATA DIRECTORY> option ignored
 INSERT INTO t VALUES(1);
 # xtrabackup backup
 # xtrabackup prepare
```